src/Installation/guides/source.md

```diff
@@ -25,9 +38,9 @@ make
 
 To install Vinegar (in the source directory of vinegar):
 ```sh
-make PREFIX=/usr install
+sudo make PREFIX=/usr install
 ```
-This will install Vinegar itself and it's icons for Roblox and Vinegar. It will also install the desktop files required for recognition by MIME.
+In this example we will use the /usr directory so of course we will need root permissions, so sudo is needed (you can change the directory if wanted, though this is the recommended directory for the installation). This will install Vinegar itself and it's icons for Roblox and Vinegar. It will also install the desktop files required for recognition by MIME.
```

Comment:
```suggestion
In this example we will install Vinegar on the system, which requires the `/usr` prefix in-order to be recognized by other applications, this will install Vinegar itself and it's icons for Roblox and Vinegar. It will also install the desktop files required for recognition by MIME.
```